prov/verbs/src/verbs_info.c

```diff
@@ -1364,6 +1369,15 @@ int vrb_init_info(const struct fi_info **all_infos)
 	}
 
 	for (i = 0; i < num_devices; i++) {
+		if (!ctx_list[i])
+		{
```

Comment:
super nit: please move { to end of line with if statement.

Reply:
> thanks -- I wish libibverbs would have trapped for this, but oh well. I actually receive a report from RedHat about a bunch of tests failing with segfaults. I wonder if they were hitting the same sort of issue.

You're welcome! FYI, here are some details about the environment where the problem was observed:
```
$ ofed_info | head
MLNX_OFED_LINUX-5.1-0.6.6.0 (OFED-5.1-0.6.6)
```

```
$ sudo lsb_release -a
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 20.04.1 LTS
Release:	20.04
Codename:	focal
```

 ```
$ uname -a
Linux xxx 5.4.0-45-generic #49-Ubuntu SMP Wed Aug 26 13:38:52 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux
```

Reply:
Thanks. It's fixed in the last revision of the patch.